final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,61 @@
+import feedparser
+from pprint import pprint
+from myargparse import *
+from bs4 import BeautifulSoup 
+import json
+
+
+def parse(url):
```

Comment:
I'm not sure that you have to add this function. All it does - return result of another function.